packages/shared/src/logger/scopes/update/index.ts

```
@@ -2,9 +2,11 @@ import { BaseScope } from '../../base/baseScope';
 import { EScopeName } from '../../types';
 
 import { AppScene } from './scenes/app';
-
+import { FirmwareScene } from './scenes/firmware';
```

ESLint

[error] Expected 1 empty line after import statement not followed by another import.

(import/newline-after-import)